superset/viz.py

```diff
@@ -260,9 +260,15 @@ def query_filters(self, is_having_filter=False):
             if not (col and vals):
                 continue
             elif col in self.datasource.filterable_column_names:
-                # Quote values with comma to avoid conflict
-                vals = ["'{}'".format(x) if "," in x else x for x in vals]
```

Comment:
This should work fine too:
vals = ["'{}'".format(x) if "," in x else "{}".format(x) for x in vals]